test/integration/integration_test.go

```diff
@@ -81,8 +81,8 @@ func FeatureContext(s *godog.Suite) {
 
 	// steps to execute minishift commands
 	s.Step(`Minishift (?:has|should have) state "([^"]*)"`, minishift.shouldHaveState)
+	s.Step(`executing "minishift ([^"]*)" (succeeds|fails)$`, executingMinishiftCommandSucceedsOrFails)
```

Comment:
@hferentschik My understanding was that it matches only if it matches whole string, not just its part. It does not. So I guess it will be better to add `^` and `$` to every step matching? WDYT?